src/LanguageServer/Test/CompletionTests.cs

```diff
@@ -123,6 +123,63 @@ class oar(list):
                 .And.HaveInsertTextFormat(InsertTextFormat.PlainText);
         }
 
+        [DataRow(PythonLanguageVersion.V37)]
+        [DataRow(PythonLanguageVersion.V36)]
+        [DataTestMethod]
```

Comment:
Up to, but you can just use `LatestAvailable3X`

Reply:
Ok